CMakeLists.txt

```diff
@@ -173,6 +177,17 @@ if(BUILD_OPENCV_CAMERA_PLUGIN)
     list(APPEND OPENCV_MODULES_USED ${OPENCV_CAMERA_EXTRA_LIBS})
 endif()
 
+if(BUILD_VIDEOTRACKER_PLUGIN)
+    set(VIDEOTRACKER_EXTRA_LIBS
+        opencv_imgproc
+        opencv_features2d
+        opencv_calib3d
```

Comment:
I chased down this path briefly, but didn't get very far.  Judging from later commits, it looks like you got further down the path than I did.